BUILDER_DEV.md

```diff
@@ -0,0 +1,260 @@
+# Builder Services Development Environment
+
+## Overview
+This document outlines the steps to start and run a Builder environment for development. The builder environment includes the builder services, as well as the depot web site.
+
+## Pre-Reqs
+1. Use a Linux OS - either native or VM.
+2. Clone the habitat repo to your local filesystem. (If using a VM, you should use the local filesystem for the hab repo instead of using a mounted filesystem, otherwise things may fail later when running services).
+3. Ensure you have a Github auth token. Create from the Github site if you don't have one already. Token Capabilities needed: read:org, user:email
+4. The sample commands below use the 'httpie' tool. Install it if not present on your system (https://github.com/jkbrzt/httpie).
+5. A recent version of the habitat cli should also be installed in your dev environment (https://www.habitat.sh/docs/get-habitat/)
+
+## Bootstrap the OS with required packages
+
+You need to make sure you have the required packages installed.
+Run the appropriate shell script under the `support/linux` folder to install the packages. E.g.:
+```
+sh support/linux/install_dev_0_ubuntu_latest.sh
+sh support/linux/install_dev_9_linux.sh
```

Comment:
We should link to the [BUILDING.md](https://github.com/habitat-sh/habitat/blob/master/BUILDING.md#ubuntu-latest-1610yakkety) as part of the prerequisites to ensure this is always accurate rather than duplicate the documentation for installing the habitat dev toolchain.